examples/json-type/json_type.md

```diff
@@ -10,7 +10,7 @@ The `json` type can be explained as a union of the simple basic types, `string`,
 
 ## Related links
 - [Access JSON elements](/learn/by-example/access-json-elements/)
-- [Converting to user-defined type](/learn/by-example/converting-to-user-defined-type/)
+- [Converting JSON with langlib functions](/learn/by-example/converting-json-with-langlib-functions/)
```

Comment:
Dont we need to say what we convert the jsons to?

Reply:
Changed